pkg/integrations/github/snippet-linux.sh

```diff
@@ -0,0 +1,8 @@
+mkdir ~/actions-runner && cd ~/actions-runner
+curl -o actions-runner-linux.tar.gz -L {{ .RunnerURL }}
+tar xzf ./actions-runner-linux.tar.gz
+sudo ./bin/installdependencies.sh
+./config.sh --token {{ .Token }} --url {{ .RepoURL }} --name {{ .Name }} --unattended --replace --labels {{ .Labels }}
+sudo ./svc.sh install
+chcon system_u:object_r:usr_t:s0 $(pwd)/runsvc.sh
+sudo ./svc.sh start
```

Comment:
Did you check if this starts the cli as the current user and not as root?

Reply:
this i need to verify, it creates a systemd service, but most probably its not a user service

Reply:
Ok otherwise you can try to use same approach as for cirrus

Reply:
it automatically adds the `User` key to the service file it creates, so its running as the user..

```
[fedora@ip-10-0-2-128 ~]$ cat /etc/systemd/system/actions.runner.anjannath-crc.mapt8c0baf8f.service
[Unit]
Description=GitHub Actions Runner (anjannath-crc.mapt8c0baf8f)
After=network.target

[Service]
ExecStart=/home/fedora/actions-runner/runsvc.sh
User=fedora
WorkingDirectory=/home/fedora/actions-runner
KillMode=process
KillSignal=SIGTERM
TimeoutStopSec=5min

[Install]
WantedBy=multi-user.target
```